server/db/models/Venue.js

```diff
@@ -6,7 +6,36 @@ const {
 const db = require('../db');
 const { sequelize } = require('./Order');
 
-class Venue extends Model {}
+class Venue extends Model {
+  authenticate = async function ({ email, password }) {
+    const venue = await Venue.findOne({
+      where: {
+        email,
+      },
+    });
+    if (venue && (await bcrypt.compare(password, venue.password))) {
+      const token = jwt.sign({ id: venue.id }, process.env.ACCESS_TOKEN_SECRET);
+      return token;
+    }
+    const error = Error('bad credentials');
+    error.status = 401;
+    throw error;
+  };
+  byToken = async function (token) {
+    try {
+      const { id } = await jwt.verify(token, process.env.ACCESS_TOKEN_SECRET);
+      const venue = await Venue.findByPk(id);
+      if (venue) return venue;
+      const error = Error('bad credentials');
+      error.status = 401;
+      throw error;
+    } catch (ex) {
+      const error = Error('bad credentials');
+      error.status = 401;
+      throw error;
+    }
+  };
+}
```

Comment:
I realize now that these need to be updated to match how I modified the User methods